dwh/query_lib_big_query.py

```diff
@@ -0,0 +1,325 @@
+"""Module that implements Query Engine for BigQuery."""
+import typing as tp
+
+import pandas as pd
+
+from base import EncounterConstraints, ObsConstraints, PatientQuery
+
+
+_DATE_VALUE_SEPARATOR = ","
+
+try:
+    from google.cloud import bigquery
```

Comment:
As we discussed, I don't mind requiring `bigquery` API as a dependency. I am not sure that even with this the tests pass if `bigquery` is not installed (as there are BQ specific tests too). One can make the same argument for Spark too. I suggest we drop this and simply import `bigquery`.